app/ide-desktop/lib/dashboard/src/authentication/src/components/app.tsx

```diff
@@ -0,0 +1,117 @@
+/** @file Main App module responsible for rendering virtual router. */
```

Comment:
what is virtual router? Please add this info to the docs.

Reply:
Added